### DISCUSSION TITLE
【2023.8.24】community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#927] [meeting]
##### Meeting replay: [Bilibili](https://www.bilibili.com/video/BV1zF411C7Hz/?spm_id_from=333.999.0.0)

**Conference link：**
> **Note**: Every two weeks at 9:00 p.m
https://meeting.tencent.com/dm/y6D9zgQbcXmF 📱#Tencent Conference：333-9699-4866

This biweekly will be a long awaited mega-merge and the beginning of our `OpenIM:3.2.0` merge

**Outline of the meeting:**
- [User case studies](https://github.com/OpenIMSDK/community/blob/main/ADOPTERS.md) and [Register the use of OpenIM](https://github.com/OpenIMSDK/Open-IM-Server/issues/379)
- This section describes the OpenIMSDK project
- What are we doing now
- Overall project planning and [roadmap for OpenIM](https://github.com/orgs/OpenIMSDK/projects/4/views/2)
- We also have Slack channels for you to communicate and discuss. To join, visit https://slack.com/ and join our [👀 Open-IM-Server slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) team channel.
- [Contribution guide](https://github.com/OpenIMSDK/Open-IM-Server/blob/main/CONTRIBUTING.md), describes the entire contribution step
- Summary of the meeting, [summary of the document](https://docs.google.com/document/d/1nx8MDpuG74NASx081JcCpxPgDITNTpIIos0DS6Vr9GU/edit?usp=sharing)

**If you have a topic to discuss, please comment below.**

**Comment by IRONICBo** [comment]
OpenKF:

1. User online and offline callbacks are out of work.

**Comment by skiffer-git** [comment]
Build the OpenIM Flutter demo from scratch

**Comment by charlie-bit** [comment]
im-server optimization

How to accurately obtain online connection information of users connected to im-server Gateway? 

now operation: need to traverse the gateway connection obtained by zk, and then further filter users.

**Comment by charlie-bit** [comment]
Question:  Mark a message as read, and when there are multiple people in the same group, will the same message be repeatedly operated?
